bootstrap3-typeahead.js

```diff
@@ -223,17 +224,17 @@
             }
 
             this.shown = true;
-            this.$element.attr('aria-expanded', 'true');
-            this.$element.attr('aria-owns', getOrGenId(element[0]));
+            this.$parent.attr('aria-expanded', 'true');
+            this.$parent.attr('aria-owns', getOrGenId(element[0]));
             return this;
         },
 
         hide: function () {
             this.$menu.hide();
             this.prevItems = [];
             this.shown = false;
-            this.$element.attr('aria-expanded', 'false');
-            this.$element.attr('aria-activedescendant', null);
+            this.$parent.attr('aria-expanded', 'false');
+            this.$parent.attr('aria-activedescendant', null);
```

Comment:
Check the [spec](https://www.w3.org/TR/wai-aria-1.1/#combobox) - `aria-activedescendant` remains on the `input`

Reply:
Think I've sorted this now